daffodil-tdml-lib/src/main/scala/org/apache/daffodil/tdml/TDMLRunner.scala

```diff
@@ -1129,23 +1186,32 @@ case class ParserTestCase(ptc: NodeSeq, parentArg: DFDLTestSuite)
         firstParseResult.getBlobPaths.foreach { blobPath =>
           Files.delete(blobPath)
         }
+        firstParseResultSax.getBlobPaths.foreach { blobPath =>
+          Files.delete(blobPath)
+        }
       }
       case OnePassRoundTrip => {
         val outStream = new java.io.ByteArrayOutputStream()
+        val outStreamSax = new java.io.ByteArrayOutputStream()
 
         doOnePassRoundTripUnparseExpectSuccess(outStream, firstParseResult, implString)
+        doOnePassRoundTripUnparseExpectSuccess(outStreamSax, firstParseResultSax, implString)
 
         // It has to work, as this is one pass round trip. We expect it to unparse
         // directly back to the original input form.
 
         VerifyTestCase.verifyUnparserTestData(new ByteArrayInputStream(firstParseTestData), outStream, implString)
+        VerifyTestCase.verifyUnparserTestData(new ByteArrayInputStream(firstParseTestData), outStreamSax, implString)
 
         // Done with the first parse result, safe to clean up blobs if there
         // was success. This won't get called on failure, which is fine--leave
         // blobs around for debugging
         firstParseResult.getBlobPaths.foreach { blobPath =>
           Files.delete(blobPath)
         }
+        firstParseResultSax.getBlobPaths.foreach { blobPath =>
+          Files.delete(blobPath)
+        }
       }
       case TwoPassRoundTrip => {
```

Comment:
It would be nice if we could reduce alot of this duplication into functions.

Speaking of duplication, was there a difference in amount of time needed to run all the tests by parsing everything twice, once for sax and one regular? If there is a noticable difference, we might want to put this behind a flag and only enable it on GitHub actions or before releases as a verification step.

Reply:
'sbt test' was only about 20 seconds longer when run with the 1763fb2d13eba1df5dacb44c560bd24010f35881D commit as the base. Will retest after the final rebase